controller/controller_server/csi_controller_server.py

```diff
@@ -594,7 +593,7 @@ def _get_object_name_and_prefix(self, request, max_name_prefix_length, max_name_
         if len(full_name) > max_name_length:
             hashed_name = utils.hash_string(name)
             full_name = self._join_object_prefix_with_name(prefix, hashed_name)
-        return full_name[:max_name_length], prefix
+        return full_name[:max_name_length]
```

Comment:
please update function name accordingly

Reply:
done

Reply:
what about other functions? e.g. `_get_volume_name_and_prefix`?

Reply:
you right, missed it. fix